lib/forms/pegasus_form_validation.rb

```diff
@@ -4,6 +4,7 @@
 require_relative 'pegasus_form_errors'
 require_relative '../cdo/regexp'
 require_relative '../cdo/geocoder'
+require_relative '../../pegasus/forms/hoc_signup_2017'
```

Comment:
In that case, I again think we should provide the validation a set of forms, and attempt to match from that. As it stands (leftover from the original design) it just tries to instantiate whatever class name you provide, which means the class name must exist in the context. The validation contract is that whatever name the caller provides should exist and have a `normalize` method.